templates: minor wording fixes (#538)
inveniosoftware/docs-invenio-rdm: docs/customize/look-and-feel/templates.md

There are 2 options to change this template:

> 1. Change the configuration value setting to the new template path. In this way, you can organize your template's files as you see fit.

Reviewer: ```suggestion
1. Change the configuration value setting to the new template path. This way, you can place and name your template file as you see fit.
```

> 2. Create a file with the same path relative to `templates` as the original. In this way, you can keep the template's files as structured and organized in InvenioRDM.

Reviewer: ```suggestion
2. Create a file with the same path relative to `templates` as the original. This way, you can keep the same structure and organization as InvenioRDM for your template file.
```
